lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -6114,8 +6157,79 @@ def model_get_unmet_load_hours(model)
     return result
   end
 
+  # A function to check whether a space is a return / supply plenum.
+  # This function only works on spaces that is categorized as return or supply air plenum
+  # For zones works as plenum but not correctly categorized in the OS will not be identified by this function
```

Comment:
Unless I am missing something, this function doesn't seem to look if a space has been categorized as a plenum but only looks the type of return plenums that the space be might used for.

Reply:
I will update the wording based on the one written in the document.